### PR TITLE
Make rmarkdown files work with styler and lintr

### PR DESCRIPTION
**Pull request by tvatter**
See #2450

### CODE REVIEW

**Comment by cjber** [comment]
I have been working with rnoweb (.rnw) files recently and they work basically the same as R Markdown so might be worth adding as well if you were happy to.

**Comment by tvatter** [comment]
I don't agree. For rnoweb files, I would rather have the linters/fixers related to latex.

**Comment by cjber** [comment]
Ah of course, I do agree, hadn't thought about that

File: autoload/ale/linter.vim [hunk 1]
```diff
@@ -13,6 +13,7 @@ let s:default_ale_linter_aliases = {
 \   'Dockerfile': 'dockerfile',
 \   'csh': 'sh',
 \   'plaintex': 'tex',
+\   'rmarkdown': 'r',
```
Comment (w0rp) [inline autoload/ale/linter.vim]: Update the documentation as mentioned above, and I will merge this.

Comment (tvatter) [reply autoload/ale/linter.vim]: Added as you suggested, I think that the travis failure is outside of my control:

https://travis-ci.com/w0rp/ale/jobs/206776131#L450

**Comment by w0rp** [comment]
Cheers! :beers:

**Comment by tvatter** [comment]
Thanks!

**Pull request merged**